ocp_resources/node_network_configuration_policy.py

```diff
@@ -345,6 +345,12 @@ def status(self):
                 return condition["reason"]
 
     def wait_for_configuration_conditions_unknown_or_progressing(self, wait_timeout=30):
+        if (
```

Comment:
This function wait for unknown_or_progressing
it doesn't make sense to return when SUCCESSFULLY_CONFIGURED